dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCOracle.scala

```diff
@@ -19,9 +21,10 @@ import org.bitcoins.keymanager.{DecryptedMnemonic, WalletStorage}
 import java.time.Instant
 import scala.concurrent.{ExecutionContext, Future}
 
-case class DLCOracle(private val extPrivateKey: ExtPrivateKeyHardened)(implicit
+class DLCOracle(private[this] val extPrivateKey: ExtPrivateKeyHardened)(implicit
```

Comment:
Good idea on `private[this]` , is there a reason for going from `case class ` -> `class` ?

Reply:
Can't have `private[this]` in a `case class`